index.js

```diff
@@ -67,7 +67,13 @@ Workload.prototype.stop = function stop () {
 Workload.prototype._visit = function _visit (req) {
   var self = this
 
-  req.headers = xtend(req.headers, this._defaultHeaders, {'user-agent': USER_AGENT})
+  req.headers = xtend(req.headers, this._defaultHeaders)
```

Comment:
You can just put the user-agent object as the first argument to `xtend`, then it will be overwritten if any of the two other arguments set it. That way you don't need to manually check if it's undefined.

Reply:
I'll optimize this later 😃